junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/EnabledForJreRange.java

```diff
@@ -102,9 +104,36 @@
 	 * possible version.
 	 *
 	 * @see JRE
+	 * @see #maxFeatureVersion()
 	 */
 	JRE max() default JRE.OTHER;
 
+	/**
+	 * Java Runtime Environment feature version which should be used as the lower
+	 * boundary for the version range that determines if the annotated class or
+	 * method should be enabled.
+	 *
+	 * <p>Defaults to {@code -1} to signal that {@link #min()} should be used instead.
+	 *
+	 * @since 5.12
+	 * @see #min()
```

Comment:
Might be useful to `@see JRE#featureVersion()` here to connect the two in the reader's mind.

Reply:
Indeed. I'll improve the Javadoc once we've determined if we want to implement this feature.